pymemcache/serde.py

```diff
@@ -63,13 +64,22 @@ def python_memcache_serializer(key, value):
     else:
         flags |= FLAG_PICKLE
         output = BytesIO()
-        pickler = pickle.Pickler(output, PICKLE_VERSION)
+        pickler = pickle.Pickler(output, pickle_version)
         pickler.dump(value)
         value = output.getvalue()
 
     return value, flags
 
 
+def get_python_memcache_serializer_pickle_version(pickle_version=None):
```

Comment:
I think this could be shortened to `get_python_memcache_serializer()`, in case there are other reasonable parameters to provide in the future. That will keep the public API more stable.

Reply:
good idea, I am bad at naming things.